### PR TITLE
Release of 0.3.0

### PR DESCRIPTION
**Pull request by myungjoo**
Please do SR with this.

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

If you have PRs to be merged before 0.3.0, please hurry up! :)

### CODE REVIEW

**Comment by myungjoo** [comment]
I'll resume this when nnstreamer becomes 5.5 m2 rc-x ready.

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,14 @@
+0.2.0 -> 0.3.0:
+	- Tizen Public C-API Single/Pipeline RC1 Fixed for 5.5 M2
```
Comment (leemgs) [inline CHANGES]: I want you to append a document on the major changes.
* https://github.com/nnsuite/nnstreamer/tree/master/Documentation

File: packaging/nnstreamer.spec [hunk 2]
```diff
@@ -14,7 +14,7 @@ Summary:	gstremaer plugins for neural networks
 # 2. Tizen  : ./packaging/nnstreamer.spec
 # 3. Android: ./jni/nnstreamer.mk
 # 4. Meson  : ./meson.build
-Version:	0.2.1
+Version:	0.3.0
```
Comment (leemgs) [inline packaging/nnstreamer.spec]: Looks good.

File: CHANGES (hunk 1 above)
Comment (myungjoo) [reply CHANGES]: Uh.. let's update Documentation in another thread.

Comment (myungjoo) [reply CHANGES]: Addressed in #1638

**Pull request merged**